src/pages/RegistrationFlow/LogoRF/LogoRF.tsx

```diff
@@ -0,0 +1,30 @@
+import "./LogoRF.scss";
+
+export const LogoRF = () => {
```

Comment:
can you export a single image instead of placing 4 of them?

Reply:
fixed 3b34439